packages/cli/src/commands/start.ts

```diff
@@ -65,7 +65,10 @@ const command: CommandModule = {
     if (typeof memcachedInfo === 'string') throw new Error('UNIX domain socket is unexpected')
 
     // run start command with env
-    const appProcess = exec(dappConfig.startAppCommand, {
+    const startAppCommandSplitted = dappConfig.startAppCommand.split(' ')
```

Comment:
Change Request: use [shlex](https://www.npmjs.com/package/shlex) library to parse command line arguments.

Reply:
Used shlex.